auth/auth.go

```diff
@@ -52,8 +52,20 @@ func ConfigureGithubClient() GithubClient {
 
 	tc := oauth2.NewClient(context.Background(), ts)
 
+	var githubClient *github.Client
+
+	if os.Getenv("GITHUB_HOSTNAME") != "" {
```

Comment:
nit: can we change the environment variable name to something like: `GITHUB_URL` and expect the user to pass in the full URL? This would potentially remove confusion from users whether they would have to include `https:` at the front or not.